project-base/kubernetes/deployments/webserver-php-fpm.yml

```diff
@@ -55,7 +55,7 @@ spec:
                     subPath: parameters.yml
             -   name: modify-docker-image
                 image: ~
-                command: ["sh", "-c", "cd /var/www/html && ./phing db-create db-demo"]
+                command: ["sh", "-c", "cd /var/www/html && ./phing db-create"]
```

Comment:
it is also needed to run indexes recreate and products-export from microservices since we do not have persisted elasticsearch but these needs to be running befor that phing target